exporter/signalfxexporter/translation/constants.go

```diff
@@ -119,14 +143,43 @@ translation_rules:
 
 - action: copy_metrics
   mapping:
-    cpu.idle: machine_cpu_cores
```

Comment:
Why is cpu.idle getting mapped to number cores? Isn't it converted in previous rule?

Reply:
it's a step to prepare for computation of `cpu.num_processors` in the next rule

Reply:
let me put a comment here